tools/extra/pac/fpgabist/bist_common.py

```diff
@@ -35,6 +35,8 @@
 REQ_CMDS = ['lspci', 'fpgainfo', 'fpgaconf', 'fpgadiag', 'fpga_dma_test',
             'bist_app']
 
+ROOT_PATH = '/sys/class/fpga'
+FPGA_DEVICE = os.path.join(ROOT_PATH, 'intel-fpga-dev.{instance_id}')
```

Comment:
This doesn't seem to be anyhow connected to this change.
Am I missing something?

Reply:
Good catch. this was unnecessary. I had started to do a cleanup will address in later PR